src/builder/commands/pip.rs

```diff
@@ -326,6 +326,12 @@ fn version_req(hash: &mut Digest, fname: &Path) -> Result<(), VersionError> {
                 if chunk.len() == 0 || chunk.starts_with("#") {
                     continue;
                 }
+                if chunk.starts_with("-r ") {
+                    if let Some(req) = chunk.rsplit(" ").next() {
+                        version_req(hash,
+                            fname.parent().unwrap().join(req).as_path());
```

Comment:
It's usually enough `&something.join(x)` instead of `somethign.join(x).as_path()`